### PR TITLE
Improve TLS.md configuration instructions

### PR DESCRIPTION
**Pull request by pshankinclarke**

### CODE REVIEW

File: TLS.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ To connect to this Valkey server with `valkey-cli`:
         --key ./tests/tls/valkey.key \
         --cacert ./tests/tls/ca.crt
 
-This will disable TCP and enable TLS on port 6379. It's also possible to have
+Specifying `port 0` will disable TCP and enable TLS on port 6379. It's also possible to have
```
Comment (zuiderkwast) [inline TLS.md]: Just specifying `port 0` doesn't enable TLS on port 6379.

"This" referred to the whole example above, with `--tls-cert-file` and all that.

```suggestion
Specifying `port 0` will disable TCP. It's also possible to have
```

Comment (pshankinclarke) [reply TLS.md:61]: Thank you for the clarification, @zuiderkwast, I was searching for an explanation for `port 0`. My change was based on the configuration described in the  [TLS Redis Docs](https://redis.io/docs/latest/operate/oss_and_stack/management/security/encryption/) Attached is a screenshot :
 
Captura de pantalla 2024-05-02 a las 9 37 05 a  m 
Let me know if there's a way to incorporate this in TLS.md.

Comment (pshankinclarke) [reply TLS.md]: Thanks for the clarification, your suggestion looks good.

**Review by zuiderkwast — APPROVED** [review]

**Comment by zuiderkwast** [comment]
You need "Signed-off-by" on the suggestion too.  You can do `git rebase HEAD~1 --signoff` and the force-push.

**Comment by zuiderkwast** [comment]
Thanks!

Actually, I think we should move this information to https://github.com/valkey-io/valkey-doc/blob/main/topics/encryption.md, where we have all the other documentation that will also be on the website (some time soon).

**Comment by zuiderkwast** [comment]
I see now that it's actually the same text that has diverged...

**Pull request merged**